mods/d2k/maps/worm-graveyard/destroyable-tiles.yaml

```diff
@@ -0,0 +1,198 @@
+^new_tile_undestroyable:
+	Inherits@1: ^ExistsInWorld
+	Inherits@2: ^SpriteActor
+	FrozenUnderFog:
+	WithSpriteBody:
+	CombatDebugOverlay:
+	Building:
+		TerrainTypes: Rock, Concrete, Cliff, Sand, Rock, Transition, Spice, SpiceSand, Dune,
+	RenderSprites:
+		Palette: terrain
+	Health:
+		HP: 20000
+	Armor:
+		Type: invulnerable
+	HitShape:
+		UseTargetableCellsOffsets: false
+		Type: Rectangle
+			TopLeft: -512, -512
+			BottomRight: 512, 512
+	MapEditorData:
+		Categories: System
+	QuantizeFacingsFromSequence:
+	Interactable:
+	Tooltip:
+		GenericName: pasage
+		Name: passage
+		ShowOwnerRow: false
+
+^new_tile_destroyable:
+	Inherits: ^new_tile_undestroyable
+	Targetable:
+		TargetTypes: Ground, Structure
+		RequiresForceFire: true
+	Armor:
+		Type: wall
+	Health:
+		HP: 30000
+	Tooltip:
+		GenericName: pasage (Destroable)
+		Name: passage (Destroable)
+	OwnerLostAction:
+		Action: ChangeOwner
+		Owner: Neutral
+	HitShape:
+		UseTargetableCellsOffsets: true
+	Explodes:
+		Type: Footprint
+		Weapon: cliffExplode
+		EmptyWeapon: cliffExplode
+	MapEditorData:
+		Categories: Destroable_tiles
+
+^new_tile_destroyed:
+	Inherits: ^new_tile_undestroyable
+	CaptureManager:
+	Building:
+	OwnerLostAction:
+		Action: ChangeOwner
+		Owner: Neutral
+	Tooltip:
+		GenericName: pasage (repairable)
+		Name: passage (repairable)
+	CaptureManager:
+	CapturableProgressBar:
+	CapturableProgressBlink:
+		Interval: 30
+	Capturable:
+		Types: cliff
+	HitShape:
+		UseTargetableCellsOffsets: true
+		Type: Rectangle
+			TopLeft: -1048, -1048
+			BottomRight: 1024, 1054
+	Targetable:
+		TargetTypes: Ground, Structure
+		RequiresForceFire: true
+	MapEditorData:
+		Categories: Destroable_tiles
+
+^CliffAvalanche:
+	ExternalCondition@rockAvalanche:
+		Condition: rock_avalanche
+	KillsSelf@rockAvalanche:
+		RequiresCondition: rock_avalanche
+		DamageTypes: SmallExplosionDeath
+
+^Infantry:
+	Inherits@avalancheKill: ^CliffAvalanche
+
+^Vehicle:
+	Inherits@avalancheKill: ^CliffAvalanche
+
+engineer:
+	Captures@cliff:
+		CaptureTypes: cliff
+		ConsumedByCapture: false
+		CaptureDelay: 150
+		ValidRelationships: Enemy, Neutral, Ally
+
+
+
```

Comment:
this triple whitespace is pretty weird, we don't do this. There's also some double whitespaces in this file